### DISCUSSION TITLE
Dependabot - what's working and not working for you?

### DISCUSSION THREAD

**Discussion by naomi789** [community/community#42343] [Code Security]
### Select Topic Area

Product Feedback

### Body

TL;DR
Fill out the survey and you may be selected for a 60 minute interview on Zoom about Dependabot and receive a $125 incentive!

Hi there,
GitHub is in the early stages of exploratory research to understand who uses Dependabot and who does NOT use Dependabot. If you're willing to chat for 60 minutes on zoom some time this month, please fill out this two minute, nine question multiple choice survey. I'm looking to chat with another 3-5 people. FYI, there aren't any "wrong" answers on the survey, you qualify no matter what your job title is, how much you use Dependabot, even if you don't know what Dependabot is, etc :) There's no need to prepare anything in advance. In recognition of your time for the 60 minute interview, we'll send you a $125 gift card/ cash card after the interview.

Link to survey: https://github.surveymonkey.com/r/HVFMS33

Feel free to comment/upvote about Dependabot pain points here, too! Especially if you can't fill out the survey/don't have time for the interview.

**Comment by nategoethel** [comment]
It would be great if Dependabot could open issues in the case where it fails to open a PR!

**Comment by ghost** [comment]
For Go dependency-checks, it would be good that Dependabot uses `govulncheck` behind the scenes (see https://go.dev/security/vuln). That would avoid a lot of false alerts.

**Comment by Adminyoungmarsnet** [comment]
Sería bueno para ayudar a github con los controles de propietario y no allá tanto jakeo de cuenta
